redux-saga-state-machine/src/toXstateConfig.ts

```diff
@@ -101,25 +127,36 @@ export const toXstateConfig = (description) => {
     if (state.activities) {
       const newActivities: string[] = [];
       for (const activity of state.activities) {
-        const name = getActionName(actionsMap, activity);
+        const name = getActionOrActivityName(actionsMap, activity);
         actionsMap[name] = activity;
         newActivities.push(name);
       }
       newState.activities = newActivities;
     }
 
+    if (state.initial) {
+      // @ts-ignore
+      newState.initial = state.initial;
+    }
+
+    if (state.parallel) {
+      // @ts-ignore
+      newState.parallel = state.parallel;
+    }
+
     if (state.states) {
+      // @ts-ignore
```

Comment:
I couldn't figure out how to get Typescript to accept that the `newState` object can have `initial`, `parallel`, and `states` fields.

Any ideas on how to handle this would be greatly appreciated!

Reply:
oh i see the simple or compound business. tsc thinks its simple which has states: undefined
let me investigate